packages/opentelemetry-instrumentation-openai/opentelemetry/instrumentation/openai/shared/completion_wrappers.py

```diff
@@ -60,22 +60,23 @@ async def acompletion_wrapper(tracer, wrapped, instance, args, kwargs):
     if context_api.get_value(_SUPPRESS_INSTRUMENTATION_KEY):
         return wrapped(*args, **kwargs)
 
-    async with start_as_current_span_async(
```

Comment:
This was causing `span.is_recording()` to return false in `_abuild_from_streaming_response`, in turn causing the response attribute setter to short-circuit, and the test assertion for span content to fail